department-of-veterans-affairs/va.gov-cms
# VACMS-4139: Reorder national content.

## Description
See #4139 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/107384978-75598c00-6ac0-11eb-9e83-41760de8582b.png)
![image](https://user-images.githubusercontent.com/2404547/107384995-7b4f6d00-6ac0-11eb-91c3-550357c9c399.png)
![image](https://user-images.githubusercontent.com/2404547/107385016-82767b00-6ac0-11eb-8b9c-7bd3ce6ba783.png)
![image](https://user-images.githubusercontent.com/2404547/107385035-86a29880-6ac0-11eb-989e-ad0944f891a7.png)

## QA steps
As an administrator, go to `node/add/vamc_system_policies_page` and visually verify that the form field order is correct, per screenshots.
Complete form, save, and visually verify that the node view order is correct, per screenshots.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
